tox.ini

```diff
@@ -166,6 +166,8 @@ filterwarnings =
     # Do not cause SyntaxError for invalid escape sequences in py37.
     default:invalid escape sequence:DeprecationWarning
 pytester_example_dir = testing/example_scripts
+markers =
+    issue
```

Comment:
s/issue/gh_issue/ maybe?

Reply:
We're already using `issue`, so I'd rather leave any changes to a future PR and just standardise here.

Reply:
Well, a) I do not know how this is used actually, but b) you're touching all those lines already (and therefore it would be good to do it right away).

Reply:
> We're already using issue, so I'd rather leave any changes to a future PR and just standardise here.

I never used it myself. I think we can actually remove them in the future, unless someone steps up and mentions he/she uses it on their workflow.